crates/e2e/Cargo.toml

```diff
@@ -22,6 +22,7 @@ ink_primitives = { workspace = true, default-features = true }
 
 cargo_metadata = { workspace = true }
 contract-build = { workspace = true }
+drink = { workspace = true }
```

Comment:
Should we consider having a feature flag for this, since it is bringing in a few heavy substrate dependencies and (at the moment) is not the default backend.

Reply:
fair point, I'll hide it tomorrow

Reply:
~70 new dependencies, some of which are substrate crates (for state-machines, externalities, few pallets etc)

Reply:
https://github.com/paritytech/ink/pull/1887